src/static/css/main.css

```diff
@@ -42,7 +42,7 @@ a {
   text-decoration: underline;
 }
 a:hover {
-  text-decoration: none;
```

Comment:
@xdamman what is the intent?

Reply:
Never seen hover state of a link that removes the underline if one was present. It was really weird to notice that. Felt like a bug. Like the link is going away when hovering it. So I reused one of the blues of the design to differentiate without removing the underline.

Reply:
Ok for the disappearing underline, I never realised it was weird, but you might be right.

However, adding a color to `a:hover` is changing the color for all kind of links, including buttons and it's breaking the intended design in some places.